models/dex/dex_sandwiched.sql

```diff
@@ -0,0 +1,64 @@
+{{ config(
+        
+        schema='dex',
+        alias = 'sandwiched',
```

Comment:
i like the convenience and simplicity of one word alias, but is `sandwiches` and `sandwiched` too similar? i could see it causing minor confusion in queries. ultimately, i'm good with it, just curious if you had other ideas?

Reply:
Hmmm I think it's mostly fine since the two names are descriptive enough as is (+ imo the 2 very similar options being there should make the wizard really question which he really needs). I am open to better name options if you have any though! But yeah I am definitely a sucker for good short names haha (as long as they are descriptive enough ofc)

Reply:
i'm good with the short, concise name here